resources/functionMap.php

```diff
@@ -9262,7 +9262,7 @@
 'Redis::client' => ['mixed', 'command'=>'string', 'arg='=>'string'],
 'Redis::close' => ['bool'],
 'Redis::config' => ['string', 'operation'=>'string', 'key'=>'string', 'value='=>'string'],
-'Redis::connect' => ['bool', 'host'=>'string', 'port='=>'int', 'timeout='=>'float', 'persistent_id='=>'?string', 'retry_interval='=>'?int', 'read_timeout='=>'float'],
+'Redis::connect' => ['bool', 'host'=>'string', 'port='=>'int', 'timeout='=>'float', 'persistent_id='=>'?string', 'retry_interval='=>'?int', 'read_timeout='=>'float', 'context'=>'?array{auth?:array,stream?:array}'],
```

Comment:
Should be optional - `'context='`

Reply:
And what kind of arrays are those? Could be described precisely

Reply:
Yeah sure :)

Reply:
I've added a more precise type description for the context elements.